### PR TITLE
release(kamakura): release core-services

### PR DESCRIPTION
**Pull request by bill-mahoney**
Signed-off-by: Bill Mahoney <bill.mahoney@intel.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Comment by ernestojeda** [comment]
@bill-mahoney I think edgex-go has some new docker images related to spiffe/spire. Are we also releasing those?

```
nexus3.edgexfoundry.org:10004/security-spiffe-token-provider
nexus3.edgexfoundry.org:10004/security-spire-agent
nexus3.edgexfoundry.org:10004/security-spire-config
nexus3.edgexfoundry.org:10004/security-spire-server
```

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by ernestojeda — CHANGES_REQUESTED** [review]
See comment

**Comment by lenny-intel** [comment]
> I think edgex-go has some new docker images related to spiffe/spire. Are we also releasing those?

@bill-mahoney , @ernestojeda , yes these new security services need new docker images released.

File: release/edgex-go.yaml [hunk 1]
```diff
@@ -79,4 +80,36 @@ docker:
   - image: nexus3.edgexfoundry.org:10004/security-bootstrapper-arm64
     destination:
       - nexus3.edgexfoundry.org:10002/security-bootstrapper-arm64
-      - docker.io/edgexfoundry/security-bootstrapper-arm64
\ No newline at end of file
+      - docker.io/edgexfoundry/security-bootstrapper-arm64
+  - image: nexus3.edgexfoundry.org:10004/security-spire-server
+    destination:
+    - nexus3.edgexfoundry.org:10002/security-spire-server
+    - docker.io/edgexfoundry/security-spire-server
```
Comment (ernestojeda) [inline release/edgex-go.yaml:87]: Indentation looks off here

File: release/edgex-go.yaml [hunk 2]
```diff
@@ -79,4 +80,36 @@ docker:
...
+  - image: nexus3.edgexfoundry.org:10004/security-spire-server-arm64
+    destination:
+      - nexus3.edgexfoundry.org:10002/security-spire-server-arm64
+      - docker.io/edgexfoundry/security-spire-server-arm64
+  - image: nexus3.edgexfoundry.org:10004/security-spire-agent
+    destination:
+    - nexus3.edgexfoundry.org:10002/security-spire-agent
+    - docker.io/edgexfoundry/security-spire-agent
```
Comment (ernestojeda) [inline release/edgex-go.yaml:95]: Indentation

File: release/edgex-go.yaml [hunk 3]
```diff
@@ -79,4 +80,36 @@ docker:
...
+  - image: nexus3.edgexfoundry.org:10004/security-spire-agent-arm64
+    destination:
+      - nexus3.edgexfoundry.org:10002/security-spire-agent-arm64
+      - docker.io/edgexfoundry/security-spire-agent-arm64
+  - image: nexus3.edgexfoundry.org:10004/security-spire-config
+    destination:
+    - nexus3.edgexfoundry.org:10002/security-spire-config
+    - docker.io/edgexfoundry/security-spire-config
```
Comment (ernestojeda) [inline release/edgex-go.yaml:103]: Indentation

File: release/edgex-go.yaml [hunk 4]
```diff
@@ -79,4 +80,36 @@ docker:
...
+  - image: nexus3.edgexfoundry.org:10004/security-spire-config-arm64
+    destination:
+      - nexus3.edgexfoundry.org:10002/security-spire-config-arm64
+      - docker.io/edgexfoundry/security-spire-config-arm64
+  - image: nexus3.edgexfoundry.org:10004/security-spiffe-token-provider
+    destination:
+    - nexus3.edgexfoundry.org:10002/security-spiffe-token-provider
+    - docker.io/edgexfoundry/security-spiffe-token-provider
```
Comment (ernestojeda) [inline release/edgex-go.yaml:111]: Indentation

**Review by ernestojeda — CHANGES_REQUESTED** [review]
Just some indentation issues.

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by ernestojeda — APPROVED** [review]
LGTM

**Pull request merged**